fabtests/prov/efa/src/rdm_rnr_read_cq_error.c

```diff
@@ -74,6 +72,9 @@ static int rnr_read_cq_error(void)
 
 	cnt = total_send;
 	do {
+		struct fi_cq_data_entry comp = {0};
+		struct fi_cq_err_entry comp_err = {0};
```

Comment:
This works, but from a readability perspective it is more explicit if we declare com and comp_err at the top, and memset them to 0 in the loop.

Reply:
The libfabric convention is that the variable declaration should be at the top of the code block, it doesn't have to be in the function top. A better fix is to make the whole body of the `while` loop as a function and define the variables at the top of the function, but that is overkill IMO.